ios/use_react_native-0.63.rb

```diff
@@ -5,8 +5,13 @@
 # LICENSE file in the root directory of this source tree.
 #
 
-def include_react_native!(react_native:, target_platform:, project_root:, flipper_versions:)
+def include_react_native!(options)
+  react_native, flipper_versions, project_root, target_platform = options.values_at(
+    :path, :rta_flipper_versions, :rta_project_root, :rta_target_platform
+  )
+
   require_relative File.join(project_root, react_native, 'scripts', 'react_native_pods')
+
   use_flipper!(flipper_versions) if target_platform == :ios && flipper_versions
-  use_react_native!(:path => react_native)
+  use_react_native!(options)
```

Comment:
I'm not a Ruby expert, but: `use_react_native!` now has three args: `project_root`, `target_platform`, `options`.  Doesn't this call need to include all three?

Reply:
`use_react_native!` actually only takes in one argument here: https://github.com/facebook/react-native/blob/811ccec74e5b898e9304f86f85f390908f8d317f/scripts/react_native_pods.rb#L6. The arrow (`=>`) syntax we see is short hand for creating the hash map.

Reply:
Oh, I was confusing the local use_react_native! in ios/test_app.rb versus the use_react_native! in react-native scripts/react_native_pods.rb